### PR TITLE
Fixed cmake project and supressed warnings

### PR DESCRIPTION
**Pull request by igorkorsukov**
Fixed cmake project and supressed warnings in thirdparty libs

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue

### CODE REVIEW

**Comment by RobFog** [comment]
"warn"? Do you mean "warning" or "warnings"?

**Comment by igorkorsukov** [comment]
> "warn"? Do you mean "warning" or "warnings"?

fixed, I mean "warnings"

**Comment by RobFog** [comment]
It's also in the commit message.

**Comment by igorkorsukov** [comment]
> It's also in the commit message.

Thanks, fixed

File: importexport/CMakeLists.txt [hunk 1]
```diff
@@ -58,9 +57,14 @@ add_library (
...
 
+add_subdirectory(../thirdparty/beatroot beatroot) #for midiimport
+add_subdirectory(../thirdparty/rtf2html rtf2html) #for capella
+
 target_link_libraries(${MODULE}
     ${QT_LIBRARIES}
-    midi        # for midi import
+    rtf2html    #for capella
+    midi        #for midiimport
+    beatroot    #for midiimport
```
Comment (Jojo-Schmitz) [inline importexport/CMakeLists.txt:67]: nitpicks, but I'd prefer a space after the `#`s, so it doesn't look like a preprocessor command ;-)

Comment (igorkorsukov) [reply importexport/CMakeLists.txt]: fixed :)

Comment (Jojo-Schmitz) [reply importexport/CMakeLists.txt]: thanks!

**Pull request merged**